external/CMakeLists.txt

```diff
@@ -54,4 +54,26 @@ if (NOT ${SPIRV_SKIP_EXECUTABLES})
       set_property(TARGET ${target} PROPERTY FOLDER GoogleTest)
     endforeach()
   endif()
+
```

Comment:
We can check and `add_subdirectory` re2 here so that we can checkout re2 into `external/`.